apps/projects/app/hooks/usePathSegments.js

```diff
@@ -0,0 +1,35 @@
+import { useCallback, useMemo } from 'react'
+import { usePath } from '@aragon/api-react'
+
+const PATH_REGEX = /^\/(settings|issues)(?:\/([a-zA-Z0-9=]{24})?)?($|\?|#)/
+const SEARCH_REGEX = /\?(.+)($|#)/ // everything between ? and end-of-line or a hash sign
+
+export default function usePathSegments() {
+  const [ path, requestPath ] = usePath()
+  const [ , selectedTab, selectedIssueId ] = path.match(PATH_REGEX) || []
+  const [ , search ] = path.match(SEARCH_REGEX) || []
+
+  const selectIssue = useCallback(issueId => {
+    requestPath && requestPath(`/issues/${issueId || ''}`)
+  }, [requestPath])
+
+  const query = useMemo(() => {
+    if (!search) return {}
+
+    return search.split('&').reduce(
+      (acc, param) => {
+        const [ , key, value ] = param.match(/([a-zA-Z0-9]+)=([a-zA-Z0-9=]+)/)
```

Comment:
Good call. It was originally just `param.split('=')`, but since the value has an `=` in it, that didn't work correctly. I didn't think to change it after changing it to a regex.